### ISSUE TITLE
Consider if Tiny yolo loss function is wrong

### ISSUE THREAD

**Issue by GiangHLe** [qqwweee/keras-yolo3#637]
In yolo3/model.py, in yolo_loss function: 

`def yolo_loss(args, anchors, num_classes, ignore_thresh=.5, print_loss=False):
    '''Return yolo_loss tensor
    Parameters
    ----------
    yolo_outputs: list of tensor, the output of yolo_body or tiny_yolo_body
    y_true: list of array, the output of preprocess_true_boxes
    anchors: array, shape=(N, 2), wh
    num_classes: integer
    ignore_thresh: float, the iou threshold whether to ignore object confidence loss
    Returns
    -------
    loss: tensor, shape=(1,)
    '''
    num_layers = len(anchors)//3 # default setting
    yolo_outputs = args[:num_layers]
    y_true = args[num_layers:]
    anchor_mask = [[6,7,8], [3,4,5], [0,1,2]] if num_layers==3 else [[3,4,5], [1,2,3]]`

The anchor_mask of tiny_yolo was wrong, right? It should be [[3,4,5],[0,1,2]]

**Comment by GiangHLe** [comment]
Sorry, I asked wrong format so I will close this and ask again
